homeassistant/components/unifi/device_tracker.py

```diff
@@ -16,26 +16,28 @@
 
 LOGGER = logging.getLogger(__name__)
 
-DEVICE_ATTRIBUTES = [
+CLIENT_CONNECTED_ATTRIBUTES = [
     "_is_guest_by_uap",
     "ap_mac",
     "authorized",
     "essid",
-    "hostname",
     "ip",
     "is_11r",
     "is_guest",
-    "mac",
-    "name",
     "noted",
-    "oui",
     "qos_policy_applied",
     "radio",
     "radio_proto",
-    "site_id",
```

Comment:
Did you mean to remove site_id?

Reply:
Yes, it is unique per integration so no need to keep it